compose/foundation/foundation/src/skikoMain/kotlin/androidx/compose/foundation/gestures/cupertino/CupertinoFlingBehavior.kt

```diff
@@ -52,7 +53,11 @@ internal class CupertinoFlingBehavior(
                     initialVelocity = initialVelocity,
                 ).animateDecay(flingDecay) {
                     val delta = value - lastValue
-                    val consumed = scrollBy(delta)
+                    val consumed = try {
+                        scrollBy(delta)
+                    } catch (exception: CancellationException) {
```

Comment:
After offline talk, we agreed to change an exception type to more specific (includes throwing this new type in  `Pager`)

Reply:
Done here!